### ISSUE TITLE
Occupy White Walls (876160)

### ISSUE THREAD

**Issue by KGOrphanides** [ValveSoftware/Proton#3076]
# Compatibility Report
- Name of the game with compatibility issues: Occupy White Walls
- Steam AppID of the game: 876160

## System Information
- GPU: AMD Radeon Vega 64
- Driver/LLVM version: Mesa 19.2.0-devel - padoka PPA
- Kernel version: 4.15.0-64-generic
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/KGOrphanides/f332c09e421d2e620892412762a8f1b9
- Proton version: 3.16-9  (also tried on 4.11-5 with same symptoms)
[steam-876160.log](https://github.com/ValveSoftware/Proton/files/3638772/steam-876160.log)

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
Game launches to a black, unresponsive screen.
Alt-tabbing out and back adds a cursor that can be moved, but nothing else.
Has to be force-quit.

## Reproduction
Launch game.

**Comment by kisak-valve** [comment]
Looks like this game would benefit from support for Media Foundation maturing.

**Comment by ghost** [comment]
so far as a workaround, i've been able to have it running with this custom build of proton - https://github.com/GloriousEggroll/proton-ge-custom - i'd need some guidance to figure out what about this build works to help point towards compatibility fixes to go into the main builds, but it gets the game playable, with some freezes.

**Comment by ghost** [comment]
to be clear, i'm specifically still running [4.11-GE-1](https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/4.11-GE-1) because i haven't bothered to update it. also i do have steamVR installed, which seems to launch with the game but can be closed after.

**Comment by ptelder** [comment]
> to be clear, i'm specifically still running [4.11-GE-1](https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/4.11-GE-1) because i haven't bothered to update it. also i do have steamVR installed, which seems to launch with the game but can be closed after.

I can confirm that I'm seeing the same issue and that OWW mostly works with 4.11-GE-1. Teleporting to Bumblebee Plaza results in a cryptic Unreal Engine error with no actual info on the error. 

Using the most recent GloriousEggroll build (4.15-GE-4) results in the original blank screen on start issue you get with the official 4.11 Proton.

**Comment by ghost** [comment]
> Teleporting to Bumblebee Plaza results in a cryptic Unreal Engine error with no actual info on the error.

Yeah, ran into this too, but it seems like it depends on your graphics settings to some degree. I get this with some people's galleries in general and it sometimes works after a few tries, sometimes only if I turn settings down.... It's a bit inconsistent.
